WordPress/src/main/java/org/wordpress/android/datasets/ReaderTagTable.java

```diff
@@ -363,4 +372,31 @@ private static int minutesSinceLastUpdate(ReaderTag tag) {
         Date dtNow = new Date();
         return DateTimeUtils.minutesBetween(dtUpdated, dtNow);
     }
+
+    public static void deleteRecommendedTagsAddedAsFollowedTags() {
```

Comment:
I'm wondering if we could delete the out-dated data in a migration script. We could drop the whole tag table when it comes to that - it's just a cache anyway. This code will be here forever and every user will use it once at most. Wdyt? (I realize it technically wasn't introduced in this PR, but imo it's still worth considering)

Reply:
Yep, we can include it in migration script. Wasn't sure how to migrate with RI FF enabled.

Reply:
yeah, I agree that'd be quite tricky with a FF. Feel free to create the migration script ignoring the FF - we'll delete it for everyone no matter the FF. I'll create a PR for removing the FF tomorrow anyway. Thanks 🙇

Reply:
Done in f493d12